devsetup/Makefile

```diff
@@ -199,13 +199,18 @@ edpm_deploy_instance: ## Spin a instance on edpm node
 	-oc cp scripts/edpm-deploy-instance.sh openstackclient:/tmp/
 	-oc rsh openstackclient bash /tmp/edpm-deploy-instance.sh
 
+.PHONY: standalone_deploy
+standalone_deploy: export COMPUTE_CEPH_ENABLED=${EDPM_COMPUTE_CEPH_ENABLED}
+standalone_deploy: export STANDALONE=true
+standalone_deploy: export INTERFACE_MTU=${NETWORK_MTU}
+standalone_deploy:
+	$(eval $(call vars))
+	scripts/standalone.sh ${EDPM_COMPUTE_SUFFIX} ${STANDALONE_COMPUTE_DRIVER} '${EDPM_COMPUTE_ADDITIONAL_NETWORKS}'
+
 .PHONY: standalone
-standalone: export COMPUTE_CEPH_ENABLED=${EDPM_COMPUTE_CEPH_ENABLED}
-standalone: export STANDALONE=true
-standalone: export INTERFACE_MTU=${NETWORK_MTU}
 standalone: edpm_compute ## Create standalone VM
+standalone: standalone_deploy
```

Comment:
Judging by the CI run, I think `edpm_compute` and `standalone_deploy` need to be on the same line

Reply:
ack thank you @cescgina i've been chasing another issue this morning (standalone is not getting the IP address specified by config) but will update this thanks